main.go

```diff
@@ -155,8 +155,8 @@ func main() {
 		for len(raw) > 0 {
 			var b *pem.Block
 			b, raw = pem.Decode(raw)
-			if b.Type != "CERTIFICATE" {
-				continue
+			if b == nil || b.Type != "CERTIFICATE" {
+				break
```

Comment:
Was getting a nil-pointer error here when using a cert generated via `safe x509`. Fixed by skipping when b == nil. maybe we need a better check to see if raw matched what it was before to give up?